webui/react/src/components/DownloadModelPopover.tsx

```diff
@@ -0,0 +1,56 @@
+import { CopyOutlined } from '@ant-design/icons';
+import { Input, notification, Popover, Tooltip } from 'antd';
+import React, { PropsWithChildren, useCallback, useMemo, useState } from 'react';
+
+import { ModelVersion } from 'types';
+
+interface Props {
+  modelVersion: ModelVersion;
+}
+
+const DownloadModelPopover: React.FC<Props> = (
+  { children, modelVersion }: PropsWithChildren<Props>,
+) => {
+  const [ visible, setVisible ] = useState(false);
+
+  const downloadCommand = useMemo(() => {
+    return `det checkpoint download ${modelVersion.checkpoint.uuid}`;
+  }, [ modelVersion.checkpoint.uuid ]);
+
+  const handleCopy = useCallback(async () => {
+    await navigator.clipboard.writeText(downloadCommand);
```

Comment:
**suggestion:** we have a `copyToClipboard` utility function that does this and also has some backwards compatibility. It'll be good to use this so we can centralize a single point of location to update code if we want to change the behavior of it in the future.